encryption/saber/saberKEM-sha3/bench/avx2_FireSABER_skylake.yaml

```diff
@@ -0,0 +1,7 @@
+platform: 'Intel Core i7-6600U (Skylake) CPU 2.60GHz, Ubuntu, gcc v. 7.0'
```

Comment:
They are contradicting themselves. In the text, they say i7-6600, in the Table caption, they say i5-7200. What do we do about this?

Reply:
I compared the round 1 and round 2 submissions: the table caption *does* change, but the text paragraph *does not* from v1 to v2.
So I updated the information to match the table caption from v2.

Reply:
smart move :)